v3/src/components/slider/slider-component.tsx

```diff
@@ -22,6 +22,7 @@ export const SliderComponent = observer(function SliderComponent({ tile } : ITil
   const layout = useMemo(() => new SliderAxisLayout(), [])
   const {width, height, ref: sliderRef} = useResizeDetector()
   const [running, setRunning] = useState(false)
+  const [isEditingName, setIsEditingName] = useState(false)
```

Comment:
```suggestion
  const [isEditingName, setIsEditingName] = useState(false)
```
Remove unused variables.